democracylab/emails.py

```diff
@@ -30,9 +33,12 @@ class HtmlEmailTemplate:
     base_template = loader.get_template('html_email_frame.html')
     section_templates = {
         EmailSection.Header: loader.get_template('html_email_header.html'),
+        EmailSection.Headerleft: loader.get_template('html_email_headerleft.html'),
+        EmailSection.Subheader: loader.get_template('html_email_subheader.html'),
         EmailSection.Button: loader.get_template('html_email_button.html'),
         EmailSection.Paragraph: loader.get_template('html_email_paragraph.html'),
-        EmailSection.Paragraph_Center: loader.get_template('html_email_paragraph_center.html')
+        EmailSection.Paragraph_Center: loader.get_template('html_email_paragraph_center.html'),
+        EmailSection.Text_Line: loader.get_template('html_email_text_line.html'),
```

Comment:
```suggestion
        EmailSection.Text_Line: loader.get_template('html_email_text_line.html')
```

Reply:
and this is why I need to figure out how to get linting working on .py files. Thanks, good eye.